src/vs/code/electron-main/app.ts

```diff
@@ -1106,6 +1116,8 @@ export class CodeApplication extends Disposable {
 		// MCP
 		services.set(INativeMcpDiscoveryHelperService, new SyncDescriptor(NativeMcpDiscoveryHelperService));
 
+		// Browser View
```

Comment:
nit: maybe let's move this registration higher in this file, so it's closer to other web-related registrations like `// Browser Elements`, `// Web Contents Extractor` and `// Webview Manager` ?